Marlin/ultralcd.cpp

```diff
@@ -238,7 +242,7 @@ uint8_t lcdDrawUpdate = LCDVIEW_CLEAR_CALL_REDRAW; // Set when the LCD needs to
 
   #define SCREEN_OR_MENU_LOOP() \
     int8_t _menuLineNr = encoderTopLine, _thisItemNr; \
-    for (int8_t _lcdLineNr = 0; _lcdLineNr < LCD_HEIGHT; _lcdLineNr++, _menuLineNr++) { \
+    for (int8_t _lcdLineNr = 0; _lcdLineNr < LCD_HEIGHT - TALL_FONT_CORRECTION; _lcdLineNr++, _menuLineNr++) { \
```

Comment:
> TALL_FONT_CORRECTION refers to the number of pixels

Not really. TALL_FONT_CORRECTION for me is more a flag, being ether 0 or 1. Further down i use it in expressions like `+ 2*TALL_FONT_CORRECTION`.

> So this might cause a problem with Chinese.

Ahhh - not really. We have TALL_FONT_CORRECTION __because__ the Chinese font is so much 'higher' then the others, So TALL_FONT_CORRECTION is ought and used to correct problems with the Chinese font, not to make new ones.
In the special case here, Chinese is to high here to be used with 'frame' and a 5 line menu. So TALL_FONT_CORRECTION is used to reduce the lines to 4.